### PR TITLE
Credit Card and Account

### PR DESCRIPTION
**Pull request by thyerre**

### CODE REVIEW

**Comment by thyerre** [comment]
[Ver despesas por mes](https://trello.com/c/tnM5gwN7/39-ver-despesas-por-mes)

**Comment by thyerre** [comment]
[Planned expenses](https://trello.com/c/yVLY0Ayh/36-planned-expenses)

File: src/pages/CreditCard/CreditCardDetail/index.tsx [hunk 1]
```diff
@@ -26,32 +26,46 @@ import {
...
-        const dados = await getCreditCardById(id);
+        const dados = await getCreditCardById(id, month);
         setCreditCard(dados)
     }
 
     function formatDate(date: any) {
         return format(new Date(date), 'dd/MM')
     }
 
+    async function setMonthCurrent() {
```
Comment (DevKleber) [inline src/pages/CreditCard/CreditCardDetail/index.tsx:48]:
```suggestion
    async function setCurrentMonth() {
```

File: src/pages/CreditCard/CreditCardDetail/index.tsx [hunk 2]
```diff
@@ -26,32 +26,46 @@ import {
...
+        const d = new Date();
```
Comment (DevKleber) [inline src/pages/CreditCard/CreditCardDetail/index.tsx:49]:
```suggestion
        const date= new Date();
```

File: src/pages/CreditCard/CreditCardDetail/index.tsx [hunk 3]
```diff
@@ -26,32 +26,46 @@ import {
...
+        setMonthCurrent();
         getCreditCard();
     }, []);
     return (
         <Container>
              <HeaderDate>
-                <IconText name="arrow-left" size={18} color='#666666'/>
-                <TextHeaderDate>Abril</TextHeaderDate>
-                <IconText name="arrow-right" size={18} color="#666666"/>
+                <IconText name="arrow-left" size={18} color='#666666'onPress={() => {alterMonth(month > 0 ? month - 1 : 0).then(() => getCreditCard())}} />
```
Comment (DevKleber) [inline src/pages/CreditCard/CreditCardDetail/index.tsx:66]: coloca o then na funcao altermonth

**Review by DevKleber — CHANGES_REQUESTED** [review]

**Review by DevKleber — APPROVED** [review]

**Pull request merged**